homeassistant/components/influxdb.py

```diff
@@ -222,91 +223,59 @@ def influx_event_listener(event):
 
         json_body[0]['tags'].update(tags)
 
-        _write_data(json_body)
-
-    @RetryOnError(hass, retry_limit=max_tries, retry_delay=20,
-                  queue_limit=queue_limit)
-    def _write_data(json_body):
-        """Write the data."""
-        try:
-            influx.write_points(json_body)
-        except exceptions.InfluxDBClientError:
-            _LOGGER.exception("Error saving event %s to InfluxDB", json_body)
-
-    hass.bus.listen(EVENT_STATE_CHANGED, influx_event_listener)
-
-    return True
+        for retry in range(max_tries+1):
+            try:
+                influx.write_points(json_body)
+                break
+            except (exceptions.InfluxDBClientError, IOError):
+                if retry == max_tries:
+                    _LOGGER.debug(
```

Comment:
I'd like to have this at warning or error level. Having it at debug will mask perhaps fixable problems from the user.